copier/cli.py

```diff
@@ -120,7 +122,15 @@ class CopierApp(cli.Application):
         ["-n", "--pretend"], help="Run but do not make any changes"
     )
     force: cli.Flag = cli.Flag(
-        ["-f", "--force"], help="Overwrite files that already exist, without asking"
+        ["-f", "--force"],
+        help="Overwrite files that already exist, without asking. Use default answers to questions, which might be null if not specified.",
+    )
+    defaults: cli.Flag = cli.Flag(
+        ["--defaults"],
+        help="Use default answers to questions, which might be null if not specified.",
+    )
+    overwrite: cli.Flag = cli.Flag(
+        ["--overwrite"], help="Overwrite files that already exist, without asking."
```

Comment:
Please add short flags

```suggestion
    defaults: cli.Flag = cli.Flag(
        ["-l", "--defaults"],
        help="Use default answers to questions, which might be null if not specified.",
    )
    overwrite: cli.Flag = cli.Flag(
        ["-w", "--overwrite"], help="Overwrite files that already exist, without asking."
```